Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build
+FROM node:16-alpine3.15 as builder
+
+WORKDIR /app
+
+# Build tools
+RUN apk add --no-cache \
+    build-base \
+    gcc \
+    g++ \
+    make
+
+# Install dependencies
+COPY package*.json ./
+RUN npm install

```

Comment:
You should use `npm ci` instead which is much better suited to Dockerfiles. It doesn't update package-lock.json unlike `npm install`.

Reply:
@reeseovine Done, thanks for the suggestion!